acceptance/tests/provider/package/dpkg_hold_true_should_preserve_version.rb

```diff
@@ -1,4 +1,4 @@
-test_name "dpkg ensure held package should preserve version if package is already installed" do
+test_name "dpkg ensure hold package should preserve version if package is already installed" do
   confine :to, :platform => /debian-8-amd64/
```

Comment:
can you please update this to debian-9 as debian-8 was dropped in puppet-agent main